src/subscription/azext_subscription/_params.py

```diff
@@ -8,10 +8,9 @@
 
 # pylint: disable=line-too-long
 def load_arguments(self, _):
-    with self.argument_context('account subscription-definition create') as c:
+    with self.argument_context('account create') as c:
         c.argument('offer_type', required=True, help='The offer type of the subscription. For example, MS-AZR-0017P (EnterpriseAgreement) and MS-AZR-0148P (EnterpriseAgreement devTest) are available.', arg_type=get_enum_type(['MS-AZR-0017P', 'MS-AZR-0148P']))
-        c.argument('subscription_display_name', help='The subscription display name of the subscription definition.')
-
-    for scope in ['account subscription-definition create', 'account subscription-definition show']:
-        with self.argument_context(scope) as c:
-            c.argument('subscription_definition_name', options_list=['--name', '-n'], help='Name of the subscription definition.')
+        c.argument('display_name', help='The display name of the subscription.')
+        c.argument('object_id', help='The object id(s) of owners which should be granted access to the new subscription.')
+        c.argument('spn', help='The service principal names of owners which should be granted access to the new subscription.')
+        c.argument('upn', help='The user principal names of owners who should be granted access to the new subscription.')
```

Comment:
There is no help text for `--enrollment-account-name`

Reply:
Fixed

Reply:
It is fine to be like this since it is an extension command, but if you plan to convert the extension to a core command module, I would suggest fold all 3 into a single `--owners` argument and your command do the hard work by making the graph call to figure out what the real type it is. Per usage feedback, lots of users have a hard time to figure out whose graph objects, nor they have any passions to learn those terms. By just seeing the `spn`, `upn`, they might already get lost.